blueprint-metadata/src/lib.rs

```diff
@@ -280,25 +282,32 @@ fn extract_metadata() -> Metadata {
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 struct BlueprintMetadata {
     manager: BlueprintManager,
+    #[serde(alias = "mbsm_revision", alias = "master_revision", default)]
```

Comment:
I feel like just having `master_revision` would be enough. Having multiple aliases could be confusing.